R/ci.R

```diff
@@ -290,3 +290,34 @@ use_azure_coverage_badge <- function() {
   img <- glue("https://img.shields.io/azure-devops/coverage/{github_repo_spec()}/2")
   use_badge("Azure pipelines coverage status", url, img)
 }
+
+use_github_actions <- function(name = "R") {
```

Comment:
I am not really sure what `name` should be by default. This is the name of the workflow, in GitHub's [starter workflows](https://github.com/actions/starter-workflows/tree/master/ci) these are all just the name of the language, which is why I used "R" for this here. But I am open to changing it to something more specific as well.

Reply:
I think something around `R cmd check`would be useful, since we'd have different checks for different project types.